recipe/meta.yaml

```diff
@@ -8,41 +8,32 @@ source:
   url: http://download.osgeo.org/gdal/{{ version }}/gdal-{{ version }}.tar.xz
   sha256: 441eb1d1acb35238ca43a1a0a649493fc91fdcbab231d0747e9d462eea192278
   patches:
-    - clang.patch  # [osx]
-    - prepend-dll.patch  # [win]
-    - install_scripts.patch
+    - patches/clang.patch  # [osx]
+    - patches/prepend-dll.patch  # [win]
+    - patches/install_scripts.patch
 
 build:
-  number: 0
-  features:
-    - vc9  # [win and py27]
-    - vc14  # [win and (py35 and py36)]
+  number: 1
 
 requirements:
   build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
     - python
     - m2w64-xz  # [win]
-    - setuptools
-    - cmake  # [win]
-    - numpy 1.11.*
-    - libgdal {{ version }}
-    - vc 9  # [win and py27]
-    - vc 14  # [win and (py35 and py36)]
+    - pip
+    - numpy
+    - {{ pin_compatible('libgdal', exact=True) }}
```

Comment:
Change this to just `libgdal` and keep the `pin_compatible` for `run`

Reply:
I already did "manually" but left the jinja there afterwards. Still got `libspatialite` from defaults though.